src/scss/tom-select.scss

```diff
@@ -13,84 +13,86 @@
  *
  */
 
-
 // base styles
-$select-ns:										'ts' !default;
-$select-font-family:							inherit !default;
-$select-font-smoothing:							inherit !default;
-$select-font-size:								13px !default;
-$select-line-height:							18px !default;
-
-$select-color-text:								#303030 !default;
-$select-color-border:							#d0d0d0 !default;
-$select-color-highlight:						rgba(125,168,208,0.2) !default;
-$select-color-input:							#fff !default;
-$select-color-input-full:						$select-color-input !default;
-$select-color-disabled:							#fafafa !default;
-$select-color-item:								#f2f2f2 !default;
-$select-color-item-text:						$select-color-text !default;
-$select-color-item-border:						#d0d0d0 !default;
-$select-color-item-active:						#e8e8e8 !default;
-$select-color-item-active-text:					$select-color-text !default;
-$select-color-item-active-border:				#cacaca !default;
-$select-color-dropdown:							#fff !default;
-$select-color-dropdown-border:					$select-color-border !default;
-$select-color-dropdown-border-top:				#f0f0f0 !default;
-$select-color-dropdown-item-active:				#f5fafd !default;
-$select-color-dropdown-item-active-text: 		#495c68 !default;
-$select-color-dropdown-item-create-text:		rgba(red($select-color-text), green($select-color-text), blue($select-color-text), 0.5) !default;
+$select-ns:																			'ts' !default;
+$select-font-family:														inherit !default;
+$select-font-size:															13px !default;
+$select-line-height:														18px !default;
+$select-color-text:															#303030 !default;
+$select-color-border:														#d0d0d0 !default;
+$select-color-highlight:												rgb(125 168 208 / 20%) !default;
+$select-color-input:														#fff !default;
+$select-color-input-full:												$select-color-input !default;
+$select-color-disabled:													#fafafa !default;
+$select-color-item:															#f2f2f2 !default;
+$select-color-item-text:												$select-color-text !default;
+$select-color-item-border:											#d0d0d0 !default;
+$select-color-item-active:											#e8e8e8 !default;
+$select-color-item-active-text:									$select-color-text !default;
+$select-color-item-active-border:								#cacaca !default;
+$select-color-dropdown:													#fff !default;
+$select-color-dropdown-border:									$select-color-border !default;
+$select-color-dropdown-border-top:							#f0f0f0 !default;
+$select-color-dropdown-item-active:							#f5fafd !default;
+$select-color-dropdown-item-active-text: 				#495c68 !default;
+$select-color-dropdown-item-create-text:				rgba(red($select-color-text), green($select-color-text), blue($select-color-text), 50%) !default;
```

Comment:
This doesn't look to be aligned, how does the lint let them through?
Maybe there should be no tabs, only always a space, between attributes and values `attribute: value`

Reply:
They line up in the editor okay, but converted tabs to spaces.

Reply:
FYI: Here's the [editorconfig file for tom-select](https://github.com/orchidjs/tom-select/blob/master/.editorconfig). I know there are arguments for both tabs and spaces, but I prefer tabs.